lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -594,11 +597,16 @@ def write(chunk)
             unless record_json.nil?
               record = record_json
               is_json = true
+              # Extract LogEntry fields if they are present in the nested JSON.
+              # This will take precedence of the root level values.
             end
-            # Restore timestamp and severity if necessary. Note that we don't
+            # Restore these if necessary. Note that we don't
```

Comment:
nit: can we expand this comment line up to 80 characters?

Reply:
Updated.